src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -583,6 +583,19 @@ void CoilCoolingDXCurveFitPerformance::calcStandardRatings210240(EnergyPlus::Ene
             this->standardRatingIEER += IEERWeightingFactor[RedCapNum] * EERReduced;
         }
 
+        // IEER 2022 -->
+        // TODO: we can always decide and give precedence to Alternate Mode 1 or Alternate Mode 2 if present | Needs Discussion about the
+        // applicability.
+        Real64 IEER_2022(0.0);
+        Real64 EER_2022(0.0);
+        Real64 NetCoolingCapRated2022(0.0);
+        std::tie(IEER_2022, NetCoolingCapRated2022, EER_2022) =
```

Comment:
There is a lot of `std::tie` being used.  I would have welcomed structured bindings, but I don't think we use those much in the code either.  This isn't something that is executed every time step, so I don't think it's worthwhile to make changes since it is working.